svg.go

```diff
@@ -34,8 +35,21 @@ func processIcon(r io.Reader, w io.Writer) error {
 		if ok && tag.Name.Space == svgNamespace && tag.Name.Local == "svg" {
 			svgStartFound = true
 			depth++
-			if err := enc.EncodeToken(tok); err != nil {
-				return errgo.Notef(err, "cannot encode token %#v", tok)
+			st := tok.(xml.StartElement)
```

Comment:
The following is a (rather complicated) drive-by to fix the fact that SVGs in most browsers don't obey viewBox attributes when included via the path of `use -> g -> svg`. but *do* when included via the path of `use -> svg`.  To fix this, we include the icon SVG in the diagram directly, not wrapped in a `<g>`, but that means passing the icon id into `processIcon`.  This specifically comes into play with the fact that the default icon on jujucharms.com is 160x160px, but icons are 96x96px, and they will not scale otherwise, but more generally, is good practice to allow icons of any size.  Ping me if there are additional questions, but needless to say, we're simply accommodating browser behaviour here.